content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624830-b91217ae-7c79-4698-a71a-a482576e5df9.png"
+      role="presentation"
+    />
+    <Caption>Use icons without a visible label when the icon's meaning is obvious.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624831-82387f4d-29ec-4d14-b3d4-edbdc58f6257.png"
+      role="presentation"
+    />
+    <Caption>Don't use obscure icons without a visible label.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Button label text
+
+Segmented control button labels should be nouns or noun phrases that succinctly describe the choice. Button labels may not wrap to multiple lines.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624839-3ea19093-ca19-4081-b20d-70fae6c17271.png"
+      role="presentation"
+    />
+    <Caption>Phrase choice labels as nouns or noun phrases with as few characters as possible.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624840-b689b044-92d6-452b-b36f-524df912f59f.png"
+      role="presentation"
+    />
+    <Caption>Phrase choice labels as an action or with unnecessarily long strings of text.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Contextual information
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="0">
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Label
+    </Text>
+    <Text as="p" mt="0">
+      Segmented controls must have a text label, even if the label is only accessible to screen readers. However, a
+      visible text label is preferred.
+    </Text>
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Caption
+    </Text>
+    <Text as="p" mt="0">
+      An optional caption may be used to provide additional context to help users make a selection or explain how what
+      effect the selection will have. Caption text should be as short as possible.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180623806-23b98c83-e870-4eed-b1f1-635e919a3d2d.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+See the [layout](#layout) section for ways to compose a segmented control with a label.
+
+## Interaction
+
+Selections are mutually exclusive. Once a selection has been made, it can only be deselected by making a new selection.
+
+A segmented control does not require a "Save" button to apply the selection. When a choice is selected, the selection immediately affects an object, state, or view.
+
+## Layout
+
+### Composing the control with a label and caption
+
+#### Single-column
+
+<img
```

Comment:
This design is showing a two-line toolbar with labels throughout:

![image](https://user-images.githubusercontent.com/293280/182216708-1fa1a656-2250-40a3-8988-bcf8ab5e723a.png)

We don't have this anywhere on the website, and I'm afraid it's a design that is too spacious for toolbars. I think we need to validate how this would work in full-page renderings before recommending this pattern.

Reply:
Ok, I'll think of a better example to show a segmented control with the label stacked on top.

Any suggestions?